tests/test_wave_03.py

```diff
@@ -54,13 +54,14 @@ def test_friends_unique_movies_not_duplicated():
 
     # Assert
     assert len(friends_unique_movies) == 3
+    assert INTRIGUE_3 in amandas_data["friends"][0]["watched"]
```

Comment:
👍 Here, we should also ensure that the other two movies that we'd expect to have reported are also there. Essnetially, the same checks that were done for the previous test (`test_friends_unique_movies`) would apply here as well.